pkg/credentials/credentials.go

```diff
@@ -0,0 +1,35 @@
+package credentials
+
+import (
+	"path/filepath"
+)
+
+type TLSCredentials struct {
```

Comment:
might need to add comment on public struct, linter wont be happy...

Reply:
Done